README.md

```diff
@@ -1,5 +1,4 @@
 # Bootlint
-
```

Comment:
No, it's not fine; it's actually wrong. There should always be an empty line after a header.